src/components/StateBook.jsx

```diff
@@ -20,7 +20,7 @@ function Zbook({
         <div className="DivBtns">
           <button className="Comments" type="button">Comments</button>
 
-          <button className="Remove" onClick={Removebook} type="button">Remove</button>
+          <button className={Idbutton} onClick={Removebook} type="button" id="Remove">Remove</button>
```

Comment:
- [ ] Your book compnent can be better than what you, as a matter of facat, it is much more simpler than you have made it a bit complicated. See below example of what is should be 👇🏾 

```javascript
import React from "react"
import { useDispatch } from "react-redux"
import { removeBook } from "path/to/booksSlice"

function Book({ id, title, author }) {
  const dispatch = useDispatch()

  return (
    <div>
      {/* Rest of Book JSX */}
      <Button onClick={() => dispatch(removeBook(id))}>Remove</Button>
    </div>
  )
}

export default Book
```
- You can see that we are importing the **_removeBook_** action from `BookSlice` in redux. then use the displatch hook to dispatch the action when users click on the remove button.
- Also, the `Book` component should receive the ``{ id, title, author }`` as props for display.